src/legacy/core_plugins/vis_type_vislib/public/vislib/components/legend/legend.tsx

```diff
@@ -22,14 +22,24 @@ import { compact, uniq, map } from 'lodash';
 
 import { i18n } from '@kbn/i18n';
 import { EuiPopoverProps, EuiIcon, keyCodes, htmlIdGenerator } from '@elastic/eui';
+import { IAggConfig } from 'ui/agg_types';
 
 // @ts-ignore
-// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 import { createFiltersFromEvent } from '../../../../../data/public/actions/filters/create_filters_from_event';
```

Comment:
Should we re-export this from the top level of the data plugin?

Reply:
I would consider waiting for https://github.com/elastic/kibana/pull/57177

Reply:
@ppisljar as  #57177 got merged can we clean up this import? Not a blocker though